contracts/Quest.sol

```diff
@@ -136,26 +136,23 @@ contract Quest is OwnableUpgradeable, IQuest {
 
     /// @notice Checks if a Receipt token id has been used to claim a reward
     /// @param tokenId_ The token id to check
-    function isClaimed(uint256 tokenId_) public view returns (bool) {
+    function isClaimed(uint256 tokenId_) external view returns (bool) {
         return claimedList[tokenId_] == true;
     }
 
     /// @dev Returns the reward amount
-    function getRewardAmount() public view returns (uint256) {
+    function getRewardAmount() external view returns (uint256) {
         return rewardAmountInWeiOrTokenId;
     }
 
     /// @dev Returns the reward token address
-    function getRewardToken() public view returns (address) {
+    function getRewardToken() external view returns (address) {
         return rewardToken;
     }
 
-    /// @notice Allows the owner of the Quest to withdraw any remaining rewards after the Quest has ended
-    function withdrawRemainingTokens() public virtual onlyWithdrawAfterEnd {}
```

Comment:
removed this fx from the base contract because it's empty and doesn't need to be here.